CHANGELOG.md

```diff
@@ -6,6 +6,86 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # \[Unreleased\]
 
+# 20230427.061325
+
+## [holochain\_cli-0.2.0](crates/holochain_cli/CHANGELOG.md#0.2.0)
+
+## [holochain\_cli\_signal\_srv-0.2.0](crates/holochain_cli_signal_srv/CHANGELOG.md#0.2.0)
+
+## [holochain\_cli\_sandbox-0.2.0](crates/holochain_cli_sandbox/CHANGELOG.md#0.2.0)
+
+## [holochain\_cli\_bundle-0.2.0](crates/holochain_cli_bundle/CHANGELOG.md#0.2.0)
+
+## [holochain-0.2.0](crates/holochain/CHANGELOG.md#0.2.0)
+
```

Comment:
@abe-njama @ThetaSinner @jost-s 

alternatively here

Reply:
Or do we want to speak to what's included in this release - at a high level or at a detailed level.